conftest.py

```diff
@@ -122,3 +123,36 @@ def pytest_configure(config):
     #
     # Somehow the default database is always in memory, though.
     settings.DATABASES['transient']['TEST_NAME'] = ':memory:'
+
+
+@pytest.yield_fixture()
+def installed_packages_catalog():
+    with mock.patch('ideascube.serveradmin.catalog.Catalog', spec=True) as MockClass:
```

Comment:
You can use the `mocker` fixture, which avoids the top-level import.

No, it does not change anything that actually matters, just mentioning it. :smile: